tests/seta/conftest.py

```diff
@@ -116,22 +116,56 @@ def jp_index_fixture(job_priority_list):
 def fifteen_jobs_with_notes(eleven_jobs_stored, taskcluster_jobs_stored, test_user,
                             failure_classifications):
     """provide 15 jobs with job notes."""
+    counter = 0
     for job in Job.objects.all():
-        for failure_classification_id in [2, 3]:
+        counter += 1
+
+        # skip first 5 jobs related to the man-o-lantern
+        if counter < 6:
+            JobNote.objects.create(job=job,
+                                    failure_classification_id=2,
+                                    user=test_user, text="you look like a man-o-lantern")
+            continue
+
+        # add 5 jobs with raw revision, expected: 31415926535
+        if counter < 11:
+            JobNote.objects.create(job=job,
+                                   failure_classification_id=2,
+                                   user=test_user, text="314159265358")
+            continue
+
+        # Add 5 jobs with full url to revision, expected to map to 31415926535
+        if counter < 14:
+            JobNote.objects.create(job=job,
+                                   failure_classification_id=2,
+                                   user=test_user, text="http://hg.mozilla.org/mozilla-central/314159265358")
+            continue
+
+        # Add one job that doesn't parse properly and produces a revision with no data
+        if counter < 15:
             JobNote.objects.create(job=job,
-                                   failure_classification_id=failure_classification_id,
-                                   user=test_user, text="you look like a man-o-lantern")
+                                    failure_classification_id=2,
+                                    user=test_user, text="call me empty")
+            continue
+
+        # Add one job with trailing slash, expected to map to 31415926535
+        JobNote.objects.create(job=job,
```

Comment:
Since there's no 'if counter..' for this one, if there happens to be more than 15 (Job.objects) then this could potentially add more than just one more here right? How do you know for certain there are only 15 items in Jobs.objects.all()?